proposals/Layer-1.md

```diff
@@ -0,0 +1,390 @@
+# Layer 1 exception handling
+
+Layer 1 of exception handling is the MVP (minimal viable proposal) for
+implementing exceptions in WebAssembly. As such, it doesn't include higher-level
+concepts and structures. These concept and structures are introduced in later
+layers, and either:
+
+1. Improves readability by combining concepts in layer 1 into higher-level
+   constructs, thereby reducing code size.
+
+2. Allow performance improvements in the VM.
+
+3. Introduce additional new functionality not available in layer 1.
+   
+## Overview
+
+Exception handling allows code to break control flow when an exception is
+thrown.  The exeception can be any exception known by the WebAssembly module, or
+it may an unknown exception that was thrown by a called imported function.
+
+One of the problems with exception handling is that both WebAssembly and the
+host VM probably have different notions of what exceptions are, but both must be
+aware of the other.
+
+It is difficult to define exceptions in WebAssembly because (in general)
+it doesn't have knowledge of the host VM. Further, adding such knowledge to
+WebAssembly would limit the ability for other host VMs to support WebAssembly
+exceptions.
```

Comment:
For the first "host VM" here, it feels like you're talking about the JavaScript VM. Otherwise, it seems like adding exceptions to other host VMs would be no more limited than the host VM.